components/spluseins-footer.vue

```diff
@@ -41,7 +48,7 @@
 </script>
 
 <style scoped lang="scss">
-  .footer-component {
-
+  .impressum {
+    cursor: pointer;
```

Comment:
Entweder das gehört in `spluseins-impressum.vue` oder die Klasse `<span … class="impressum" …>` ist dort doppelt.